source/_integrations/osoenergy.markdown

```diff
@@ -22,6 +24,16 @@ This OSO Energy integration uses a subscription key, which a user can create for
 
 {% include integrations/config_flow.md %}
 
+## Binary sensors
+
+The **OSO Energy** integration exposes OSO Energy data as a binary sensor. It provides the following binary sensors:
+
+- Extra energy
+  - Indication if the water heater is currently in an extra energy/high demand state.
+- Power save
+  - Indication if the water heater is currently in a power save/sleep mode state.
+- Heater state
```

Comment:
Please update the name to correspond with the entity name

Reply:
Done.